app/assets/src/components/visualizations/table/base_table.scss

```diff
@@ -34,13 +34,25 @@
   text-align: center;
   text-transform: none;
 
-  .tooltipText {
-    display: inline-block;
+  .headerInner {
     max-width: 100%;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    overflow: hidden;
-    vertical-align: middle;
+    display: flex;
+    align-items: center;
+
+    .label {
+      // Shrink once you're at the maximum width (e.g. don't overflow). Do not grow unless needed.
```

Comment:
I wonder if it is better to wrap an just hide the overflow... I would have liked the ellipsis, but maybe it is better to use the space better. Maybe check with Jenn.

Reply:
I'll leave it for now, since I haven't changed the overflow behavior of the text. What this code does is make sure that if the text has ellipsis, the sort icon is still showing. Previously, the sort icon was considered part of the text and would disappear when the ellipsis shows. We can follow up on the text overflow.